src/webapp/pages/form-page/incident-action/mapIncidentActionToInitialFormState.ts

```diff
@@ -365,7 +370,7 @@ function getResponseActionSection(options: {
             {
                 id: `${responseActionConstants.verification}_${index}`,
                 label: "Verification",
-                isVisible: true,
+                isVisible: isIncidentManager ? true : false,
```

Comment:
@deeonwuli This is a required field(check like 379), so please check with Bhavana whether it is ok for a required field to be not visible for certain users.

Reply:
A requirement here is that the verification field should not be visible for users outside of the incident manager group. I will also set the value `required` property based on this condition. 

cc @bhavananarayanan

Reply:
sounds good.